QUARKUS-5614

```diff
@@ -0,0 +1,66 @@
+# QUARKUS-5614 Support @PermissionsAllowed defined on meta-annotation
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-5614
+
+Support @PermissionsAllowed defined on meta-annotation
+
+Upstream PRs:
+- https://github.com/quarkusio/quarkus/pull/43241
+
+Upsteram issue:
+- https://github.com/quarkusio/quarkus/issues/39663
+
+Documentation:
+- https://quarkus.io/guides/security-authorize-web-endpoints-reference
+
+## Scope of testing
+Verify that `@PermissionsAllowed` works correctly when applied on a meta-annotation.
+These tests will cover various scenarios of meta-annotations defined on both methods and classes.
```

Comment:
I don't think meta-annotations can work on classes, have you tried it?

Reply:
https://quarkus.io/guides/security-authorize-web-endpoints-reference#permission-meta-annotation says method or class so it is ok, my bad